CMakeLists.txt

```diff
@@ -454,6 +457,16 @@ add_compile_definitions(FOLLY_HAVE_INT128_T=1)
 set_source(folly)
 resolve_dependency(folly)
 
+# Spark qury runner depends on absl, c-ares, grpc.
```

Comment:
We should require these only when `VELOX_BUILD_TESTING` is ON.

Reply:
Updated. Thanks.